tests/src/Simple/ReadyToRunMultiModule/ReadyToRunMultiModule.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+
+:: Intentionally empty since we're just testing we can compile the framework
+:: assemblies to their own library obj files
+
+exit /b 0
```

Comment:
This script won't actually run the compiled binaries. We should execute the test to ensure we emitted correct r2r code. Take a look at tests\src\simple\ReadyToRunUnit\ReadyToRunUnit.cmd. That runs the test under the ready-to-run execution harness which collects logs of which methods were jitted instead of using their r2r native code.